Description: Try to make the build reproducible
 This patch removes the embedded compile time from libctags to try and make the
 build reproducible.
Author: James Cowgill <james410@cowgill.org.uk>
Forwarded: no
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/sdk/codelite_indexer/libctags/options.c
+++ b/sdk/codelite_indexer/libctags/options.c
@@ -928,7 +928,6 @@ static void printProgramIdentification (
 {
 	printf ("%s %s, Copyright (C) 1996-2004 %s\n",
 			PROGRAM_NAME, PROGRAM_VERSION, AUTHOR_NAME);
-	printf ("  Compiled: %s, %s\n", __DATE__, __TIME__);
 	printf ("  Addresses: <%s>, %s\n", AUTHOR_EMAIL, PROGRAM_URL);
 	printFeatureList ();
 }
